recipe/meta.yaml

```diff
@@ -11,14 +11,16 @@ source:
   folder: source
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
+  run:
+    - vs2015_runtime  # [win]
```

Comment:
Is this needed? Naively I expect it to be included automatically with `run_exports` but I'm not familiar enough with Windows to know for sure.

Reply:
Thanks for the pointer! I am new to conda build system and I don't know about the `run_exports` thing enough, but it seems the build\install still to work fine without defining the explicit dependency, so I removed them.